visualization_msgs/Marker[] markers
string[] types